src/weights.jl

```diff
@@ -1,36 +1,40 @@
-struct MetaWeights{InnerMetaGraph,U<:Real} <: AbstractMatrix{U}
-    meta_graph::InnerMetaGraph
+"""
+    MetaWeights{InnerMetaGraph<:MetaGraph,U<:Real} <: AbstractMatrix{U}
+
+Matrix-like wrapper for edge weights on a metagraph of type `InnerMetaGraph`.
+"""
+
+struct MetaWeights{InnerMetaGraph<:MetaGraph,U<:Real} <: AbstractMatrix{U}
+    g::InnerMetaGraph
 end
 
-show(io::IO, x::MetaWeights) = print(io, "metaweights")
-show(io::IO, z::MIME"text/plain", x::MetaWeights) = show(io, x)
+Base.show(io::IO, mv::MetaWeights) = print(io, "MetaWeights of size $(size(mv))")
```

Comment:
I tend to like to import things so I don't have to write `Base.` twice. But this works.

Reply:
That's one of the main things I changed. In this instance I prefer following [BlueStyle](https://github.com/invenia/BlueStyle#module-imports) recommendations:
> Prefer the use of `using` over `import` to ensure that extension of a function is always explicit and on purpose.

Reply:
Eh, sure, either way works.